website/addons/citations/provider.py

```diff
@@ -17,6 +17,14 @@ def __init__(self, provider_name):
     def serializer(self):
         pass
 
+    def check_credentials(self, node_addon):
+        valid = True
+        try:
+            node_addon.api.client()
+        except HTTPError:
```

Comment:
This error handling can be improved; only certain status codes should be allowed to pass here (probably 403).  Other status codes should still raise an error.